.ci/images_matrix.json

```diff
@@ -2,19 +2,19 @@
     "android_api": [
         {
             "api": 28,
-            "build_tools": "30.0.3"
+            "build_tools": "28.0.3"
         },
         {
             "api": 29,
-            "build_tools": "30.0.3"
+            "build_tools": "29.0.3"
         },
         {
             "api": 30,
             "build_tools": "30.0.3"
         },
         {
             "api": 31,
-            "build_tools": "32.0.0"
+            "build_tools": "30.0.3"
```

Comment:
Est-ce qu'il faudrait pas plutôt les builds 31.0.0 plutôt que pour 30.0.3 pour l'API 31 ?

Reply:
De ce que j'ai lu les build-tools 31 sont corrompus https://stackoverflow.com/questions/68387270/android-studio-error-installed-build-tools-revision-31-0-0-is-corrupted

Et sur un projet en `compileSdk 31` (qui ne précise pas version particulière de build-tools), ce sont les build-tools 30.0.3 qui vont être fetch.
Mais oui c'est pas évident, je vais rajouter un com'

Reply:
:ah: !

Ok, dans ce cas je pense que ce sera bien avec un commentaire :)